apps/heureka/src/components/filters/FilterSelect.jsx

```diff
@@ -85,16 +61,13 @@ const FilterSelect = ({ entityName, isLoading }) => {
       {activeFilters && Object.keys(activeFilters).length > 0 && (
         <Button label="Clear all" onClick={() => clearActiveFilters(entityName)} variant="subdued" />
       )}
-      {["IssueMatches", "Services"].includes(entityName) && (
-        <SearchInput
-          placeholder="Search term or regular expression"
-          className="w-96 ml-auto"
-          value={searchTerm || ""}
-          onSearch={(value) => setSearchTerm(entityName, value)}
-          onClear={() => setSearchTerm(null)}
-          // onChange={(value) => handleSearchChange(value)}
-        />
-      )}
+      <SearchInput
+        placeholder="Search term or regular expression"
+        className="w-96 ml-auto"
+        value={searchTerm || ""}
+        onSearch={(value) => setSearchTerm(entityName, value)}
```

Comment:
The same here. Think about it

Reply:
Same valuable optimization, and I implement it.